src/pages/AddPost/index.jsx

```diff
@@ -1,9 +1,17 @@
 import React from 'react'
+import NewPostHeader from '../../components/NewPost/NewPostHeader';
+import NewPostFooter from '../../components/NewPost/NewPostFooter';
+import ImageCatpion from '../../components/NewPost/ImageCaption';
+import NewPostBody from '../../components/NewPost/NewPostBody';
+
 
 export default function AddPost() {
     return (
         <div>
-            
+            <NewPostHeader/>
+            <ImageCatpion/>
+            <NewPostBody/>
+            <NewPostFooter/>
         </div>
     )
 }
```

Comment:
this is a huge logic flaw, well there is a couple.
1) move all of these components into a new folder called components inside AddPost folder in the pages (so src/pages/AddPost/components)
2) the data is stored on each component individually but let's you want to submit the data to the server, how do you do that ? how do you collect that data from all of these components? you do not have this logic here while it should be here so add it